cccat/dto.py

```diff
@@ -96,6 +96,9 @@ class CatSummary(BaseModel):
 
 class PartialUpdateCat(BaseModel):
     name: Optional[str]
+    ctime: Optional[datetime]
+    mtime: Optional[datetime]
```

Comment:
[question] Why should `mtime` and `ctime` be part of the fields that will be updated (and potentially part of the payload from the API if we ever added an "update cat endpoint")?